src/components/ImageSelector.tsx

```diff
@@ -33,7 +43,7 @@ export default function ImageSelector(): ReactElement {
           after an promotion has been successfully saved to our backend db
       */}
       {image.imageBinary && (
-        <button onClick={() => AmazonS3Service.uploadImage(image, 'promotionId123')}>
+        <button onClick={() => AmazonS3Service.uploadImage(image, 'testpromoid')}>
```

Comment:
Oh yeah we don't need this button anymore right since we're uploading the image in PromotionService.PostPromotion?

Reply:
oops I forgot to push my latest changes! but yes! I've removed it